src/pages/Login/index.js

```diff
@@ -222,6 +288,132 @@ function Login(props) {
+                          width: "40%",
+                          backgroundColor: "gray",
+                        }}
+                      ></div>
+                    </div>
+                  </Typography>
+                  <form onSubmit={handleSubmit}>
+                    <Box
+                      sx={{
+                        marginBottom: errors.username ? "40px" : "25px",
+                        marginBlock: "10px",
+                      }}
+                    >
+                      <TextField
+                        fullWidth
+                        id="username"
+                        name="username"
+                        placeholder="Username"
+                        label={user.error ? "Username" : ""}
+                        value={formData.username}
+                        onChange={handleChange}
+                        error={!!errors.username}
+                        helperText={errors.username}
+                        InputProps={{
+                          sx: {
+                            borderRadius: "8px",
+                            padding: "25px 2px",
+                            height: "56px",
+                            color: isEmpty && "red",
+                          },
+                        }}
+                      />
+                    </Box>
+
+                    <Box
+                      sx={{
+                        marginBottom: errors.password ? "40px" : "25px",
+                        marginBlock: "25px",
+                      }}
+                    >
+                      <TextField
+                        fullWidth
+                        id="password"
+                        name="password"
+                        type={"password"}
+                        placeholder="Password"
+                        label={user.error ? "Password" : ""}
+                        value={formData.password}
+                        onChange={handleChange}
+                        error={!!errors.password}
+                        helperText={errors.password}
+                        InputProps={{
+                          sx: {
+                            borderRadius: "8px",
+                            padding: "25px 2px",
+                            height: "56px",
+                            color: isEmpty && "red",
+                          },
+                        }}
+                      />
+                    </Box>
+
+                    <Grid container justifyContent="center" sx={{ mb: 3 }}>
+                      <Button
+                        type="submit"
+                        variant="contained"
+                        disabled={loading}
+                        sx={{
+                          borderRadius: "10px",
+                          padding: "18px",
+                          width: "100%",
+                        }}
+                      >
+                        Login
+                      </Button>
```

Comment:
Remove `sx={{borderRadius: "10px", padding: "18px", width: "100%", }}`  line of code and add `fullWidth` instead